src/website/app/pages/Color/RampLegend.js

```diff
@@ -0,0 +1,192 @@
+      minHeight: 100,
+      height: '50%',
+      margin: 0,
+      fontSize: theme.fontSize_mouse,
+      padding: `${theme.spacing_double} ${theme.spacing_quad}`,
+      position: 'relative'
+    },
+    '& > *::after': {
+      borderBottom: `1px solid ${theme.borderColor_danger_hover}`,
+      content: `''`,
+      position: 'absolute',
+      width: theme.spacing_triple
+    },
+    '@media(max-width: 60em)': {
+      display: 'none'
+    }
+  }),
+  hsl: {
+    textAlign: 'right',
+    '&::after': {
+      top: 15,
+      right: 0
+    }
+  },
+  mobileList: ({ theme }) => ({
+    fontSize: theme.fontSize_mouse
+  }),
+  name: ({ theme }) => ({
+    textAlign: 'right',
+    '&::after': {
+      right: 0,
+      bottom: 15
+    },
+    '& > p': {
+      position: 'absolute',
+      bottom: 0,
+      right: theme.spacing_quad
+    }
+  }),
+  white: ({ theme }) => ({
+    '&::after': {
+      bottom: 15,
+      left: 0
+    },
+    '& > p': {
+      position: 'absolute',
+      bottom: 0,
+      left: theme.spacing_quad
+    }
+  })
+};
+
+const Root = createStyledComponent('div', styles.root);
+const Black = createStyledComponent('p', styles.black);
+const Column = createStyledComponent('div', styles.column);
+const CenterColumn = createStyledComponent('div', styles.centerColumn);
+const HSL = createStyledComponent('p', styles.hsl);
+const MobileList = createStyledComponent('ol', styles.mobileList);
+const Name = createStyledComponent('div', styles.name);
+const White = createStyledComponent('div', styles.white);
+
+export default function RampLegend() {
+  const themeName = 'The name of the hue in the Mineral UI theme.';
+  const hslDefinition =
+    'The CSS Hue, Saturation & Lightness value. Hue is set by the wavelength of light, saturation is how gray the color is, and lightness is the light/dark value.';
+  const whiteA11yDefinition =
+    'The a11y rating of white on top of this color. AA and AAA denote the minimum contrast ratio.';
+  const blackA11yDefinition = 'The a11y rating of black on top of this color';
+
+  return (
+    <Root>
+      <Column>
+        <Name>
+          <p>
+            {themeName}
+          </p>
+        </Name>
+        <HSL>
+          {hslDefinition}
+        </HSL>
+      </Column>
+      <CenterColumn>
+        <ColorRampRow name={legendBkd} />
+        {getBadges()}
+      </CenterColumn>
+      <Column>
+        <White>
+          <p>
+            {whiteA11yDefinition}
+          </p>
+        </White>
+        <Black>
+          {blackA11yDefinition}
+        </Black>
+      </Column>
+      <MobileList>
+        <li>
+          <strong>{'<theme>_<value>'}</strong> {themeName}
+        </li>
+        <li>
+          <strong>White A11y Rating</strong> {whiteA11yDefinition}
+        </li>
+        <li>
+          <strong>Black A11y Rating</strong> {blackA11yDefinition}
+        </li>
+        <li>
+          <strong>hsl(H, S%, L%)</strong> {hslDefinition}
+        </li>
+      </MobileList>
```

Comment:
Nicely done. I think for small screens, though, it'd be best to display the list under the columns. Otherwise, you need to do something else to address how tight things get when they're side-by-side on a <375px wide screen.

Reply:
updated the styles to stack the list, and generally work better on a smaller screen.